tests/integrations/kitchenSink/__typegen.ts

```diff
@@ -80,10 +80,7 @@ export interface NexusGenObjects {
     // root type
     hello?: string | null // String
   }
-  OfI2: {
-    // root type
-    hello?: string | null // String
```

Comment:
@tgriesser Looks like a bug to me. If user uses `t.modify` only to augment `extensions` then why should the default backing types be altered?

Reply:
Yep, this is all pretty messy right now. Got it working temporarily, but will try and refactor this as a part of fixing #673